core/assets/bundles/bundle_ja.properties

```diff
@@ -21,6 +21,7 @@ load.map = マップ
 load.image = 画像
 load.content = コンテンツ
 load.system = システム
+load.mod = MOD
```

Comment:
Some games would write this in Katakana, but English seemed to fit the menu more, and is commonly found in Japanese games.  This is why I chose English here but Katakana within sentences below.